src/TextInput/index.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import * as React from 'react';
+import { TextInput as RNTextInput, View } from 'react-native';
+
+import { StyleSheet } from '..';
+import FormLabel from './FormLabel';
+
+type Props = {
+  +size?: 'small' | 'normal',
+  +placeholder?: string,
+  +value?: string,
+  +disabled?: boolean,
+  +onFocus?: () => void | Promise<any>,
+  +onBlur?: () => void | Promise<any>,
+  required?: boolean,
+  +label: string,
+};
+
+type State = {
+  isFocused: boolean,
+};
+
+const colors = {
+  colorTextInputDisabled: '#bac7d5',
+  colorTextInput: '#46515e',
+  borderColorInput: '#bac7d5',
+  colorPlaceholderInput: '#bac7d5',
+  borderColorInputFocus: '#0176D2',
```

Comment:
Should we not get this colors from orbit design tokens?

Reply:
At the end of the work on `TextInput` I'll take them from design tokens